app/models/book.rb

```diff
@@ -88,6 +88,75 @@ class Book < ActiveRecord::Base
   has_and_belongs_to_many :recommendations
 
   CSV_COLUMNS = ["Book Name", "ASIN"]
+  LEVELS_DICT = {
```

Comment:
I'm not really sure what this code is doing! Could you add a comment to describe it?